locales/ca_ES/LC_MESSAGES/duckduckgo.po

```diff
@@ -1432,17 +1440,17 @@ msgstr ""
 "llista raonablement gran de paraules aleatòries dels servidors DuckDuckGo."
 
 #. Form field for your email ;)
-#. From <a href="https://duckduckgo.com/newbang.html ">new !bang page</a> and <a href="https://duckduckgo.com/feedback.html">feedback page</a>. 
+#. From <a href="https://duckduckgo.com/newbang.html ">new !bang page</a> and <a href="https://duckduckgo.com/feedback.html">feedback page</a>.
 msgid "Email"
 msgstr "Correu"
 
 msgctxt "precise_user_location"
 msgid "Enable 'Sites can ask for my location.'"
-msgstr ""
+msgstr "Activeu «Els llocs poden sol·licitar la meva ubicació»."
```

Comment:
Keep the quote formatting consistent with the original and with the other translations.

```suggestion
msgstr "Activeu 'Els llocs poden sol·licitar la meva ubicació'."
```